File: 3-Bimestre/C++/senha.cpp
/*
Aluno: Carlos Eduardo Félix
3º Bimestre
Exercício 10 - Sistema simples de senha
*/
#include <iostream>
using namespace std;

int main() {
    int senha;
    const int senhaCorreta = 1234;
    int tentativas = 0;

    while (tentativas < 3) {
        cout << "Digite a senha: ";
        cin >> senha;

        if (senha == senhaCorreta) {
            cout << "Acesso liberado." << endl;
            return 0;
        }

        tentativas++;
        cout << "Senha incorreta." << endl;
    }

    cout << "Acesso bloqueado." << endl;
    return 0;
}

File: 3-Bimestre/Python/senha.py
"""
Aluno: Carlos Eduardo Félix
3º Bimestre
Exercício 10 - Sistema simples de senha
"""

senha_correta = "1234"
tentativas = 0
acesso_liberado = False

while tentativas < 3:
    senha = input("Digite a senha: ")

    if senha == senha_correta:
        acesso_liberado = True
        break

    print("Senha incorreta.")
    tentativas += 1

if acesso_liberado:
    print("Acesso liberado.")
else:
    print("Acesso bloqueado.")
